mm/2s2h/Rando/StaticData/Checks.cpp

```diff
@@ -132,6 +132,7 @@ std::map<RandoCheckId, RandoStaticCheck> Checks = {
     RC(RC_LONE_PEAK_SHRINE_LENS_CHEST,                          RCTYPE_CHEST,            SCENE_KAKUSIANA,                FLAG_CYCL_SCENE_CHEST,       0x01,                                                                RI_LENS),
     RC(RC_MAYORS_OFFICE_HP,                                     RCTYPE_NPC,              SCENE_SONCHONOIE,               FLAG_WEEK_EVENT_REG,         WEEKEVENTREG_BOMBERS_NOTEBOOK_EVENT_RECEIVED_MAYOR_HP,               RI_HEART_PIECE),
     RC(RC_MAYORS_OFFICE_KAFEIS_MASK,                            RCTYPE_NPC,              SCENE_SONCHONOIE,               FLAG_WEEK_EVENT_REG,         WEEKEVENTREG_BOMBERS_NOTEBOOK_EVENT_RECEIVED_KAFEIS_MASK,            RI_MASK_KAFEIS_MASK),
+    RC(RC_MILK_BAR_CIRCUS_LEADER_MASK,                          RCTYPE_NPC,              SCENE_MILK_BAR,                 FLAG_WEEK_EVENT_REG,         WEEKEVENTREG_50_01,                                                  RI_MASK_CIRCUS_LEADER),
```

Comment:
WEEKEVENTREG_50_01 is the flag when doing this on Day 1. Otherwise it's WEEKEVENTREG_51_80. I'm not sure how to account for that here, but currently you can only get the check on Day 1.

Reply:
ah crap, good call out. I'll probably create an actor file for this then and vb in. thanks dude!